Userland/Libraries/LibJS/Bytecode/ASTCodegen.cpp

```diff
@@ -2273,13 +2273,10 @@ static Bytecode::CodeGenerationErrorOr<void> for_in_of_body_evaluation(Bytecode:
     auto destructuring = head_result.is_destructuring;
 
     // 5. If destructuring is true and if lhsKind is assignment, then
-    if (destructuring) {
-        // a. Assert: lhs is a LeftHandSideExpression.
-        // b. Let assignmentPattern be the AssignmentPattern that is covered by lhs.
-        // FIXME: Implement this.
```

Comment:
Why are we removing these comments?

Reply:
I just blanket removed this section and then added the error back in later, but forgot to add back the comments. Thanks for catching that.